services/QuillLMS/app/views/canvas_integration/lti/launch.html.erb

```diff
@@ -1 +1,59 @@
-<%= link_to @link_text, @link_url, target: '_blank' %>
+<style>
+  .navbar { background-color: #06806B }
+
+  .logo-only {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .text-center { text-align: center }
+
+  .account-form {
```

Comment:
Let's unnest the CSS for better backwards compatibility with older browsers. https://caniuse.com/css-nesting  

e.g.
```css
.account-form {

}

.account-form .account-container {

}
...